tools/cardinal/classification.xml

```diff
@@ -1,14 +1,14 @@
-<tool id="cardinal_classification" name="MSI classification" version="@VERSION@.0">
+<tool id="cardinal_classification" name="MSI classification" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
     <description>spatial classification of mass spectrometry imaging data</description>
     <macros>
         <import>macros.xml</import>
     </macros>
     <expand macro="requirements">
-        <requirement type="package" version="2.3">r-gridextra</requirement>
-        <requirement type="package" version="3.3.5">r-ggplot2</requirement>
+        <!--requirement type="package" version="2.3">r-gridextra</requirement>
```

Comment:
We can just remove instead of comment. `git` stores it anyway.

Reply:
thanks, updated it accordingly :)